### ISSUE TITLE
journalctl pager messed up with Chinese locale

### ISSUE THREAD

**Issue by yangsheng6810** [systemd/systemd#4204]
### Submission type
- [*] Bug report
### systemd version the issue has been seen with

> 231-1

_NOTE: Do not submit bug reports about anything but the two most recently released systemd versions upstream!_
### Used distribution

> Archlinux
### In case of bug report: Expected behaviour you didn't see

Line wrapping is done correctly.

> ...
> ![correct](https://cloud.githubusercontent.com/assets/2863729/18770554/65fab4c2-8105-11e6-9524-39cd02c035e1.png)
### In case of bug report: Unexpected behaviour you saw

Line wrapping is done incorrectly. After each line that requires wrapping, there is an extra line with a single letter.

> …
> ![wrong](https://cloud.githubusercontent.com/assets/2863729/18770572/8e6a0926-8105-11e6-9018-527cc300bae0.png)
### In case of bug report: Steps to reproduce the problem

After some time, I find that locale setting is the cause. Problem can be reproduced by

> export LC_TIME=zh_CN.UTF-8
> journalctl

and can be resolved by

> export LC_TIME=en_US.UTF-8
> journalctl

I believe this is not a problem with font, as the same problem occurs in tty, where no Chinese font is available.

**Comment by stefan-it** [comment]
@yangsheng6810 In [pager.c](https://github.com/systemd/systemd/blob/master/src/shared/pager.c#L103) one of the options for `less` is `S`. E.g.:

``` bash
echo "9月 27 00:43:18 arch-64-glaukon su[1708]: pam_unix(su:session): session opened for user root by (uid=1000)" | less
```

WIll display the text with a correct line break. But when you use the less options specified in `pager.c`:

``` bash
echo "9月 27 00:43:18 arch-64-glaukon su[1708]: pam_unix(su:session): session opened for user root by (uid=1000)" | LESS=FRSXMK less
```

your output can be reproduced! So I guess the `S` option causes the problem here.

You can workaround/fix the output of `journalctl` by using the `SYSTEMD_LESS` variable + use the same options as specified for `LESS` without the `S` option:

``` bash
export SYSTEMD_LESS=FRXMK
journalctl
```

**Comment by yangsheng6810** [comment]
I checked, and find your command do reproduce the problem. It seems that this this might be a problem of `less`.

However, I found an interesting thing. My main system is Arch Linux. But when I ssh to other machines, I found the output for these distributions are all correct: Debian, gentoo, RHEL (when ssh to an arch machine, the problem remains). What can be the case? Could it be a distribution related bug?
